scripts/stitcher/README.md

```diff
@@ -0,0 +1,27 @@
+# Stitcher Benchmark
+
+This shell script is used to execute a collection of benchmarks over 24 hours that mimic a resource utilization pattern observed as part of the work done in the following paper:
+[Stitcher: Learned Workload Synthesis from Historical Performance Footprints.](https://openproceedings.org/2023/conf/edbt/paper-19.pdf), Chengcheng Wan, Yiwen Zhu, Joyce Cahoon, Wenjing Wang, Katherine Lin, Sean Liu, Raymond Truong, Neetu Singh, Alexandra M. Ciortea, Konstantinos Karanasos, Subru Krishnan. In EDBT 2023.
+If you use this workload, please reference this paper appropriately.
+
+Prior to executing the shell scripts, the data needs to be preloaded with the following commands:
+
+```sh
+java -jar benchbase.jar -b tpcc -c config/$dbms/stitcher/2023-02-24-15-11-28-957898.xml --create=true --load=true --execute=false
```

Comment:
Also, is it possible for us to have instructions for generating the config sequence given a resource utilization trace?

Reply:
I'm not sure how these were originally generated but I don't think the date matters that much? My guess is that these were the configurations of experiments that they collected telemetry for and that were then (partially) picked to mimic this one specific workload. If you look at the configurations, they have different SFs but also different query weight distributions. You would preload 4 datasets (TPC-H, TPC-C SF 16 and 160, and YCSB) and then execute these different configurations on top of those preloaded instances. This benchmark is a static snapshot of a real-world benchmark that we can publish (i.e., add to an open-source repo) because it was already used in published work. Afaik, there is some discussion as to whether they want to open-source the Stitcher code but no resolution yet.